server/views/forms/layout.njk

```diff
@@ -95,8 +95,22 @@
             {% if options.pageSubHeading %}
               <p class="page-sub-heading">{{ options.pageSubHeading }}</p>
             {% endif %}
+
             {% set defaultPageTitleClass = "govuk-body-l govuk-!-margin-0" if (getRenderedFields(form.fields, options)).length == 1 else "govuk-heading-l" %}
-            <h2 class="{{ pageTitleClass | default(defaultPageTitleClass) }}">{{ options.pageTitle }}</h2>
+
+            {% if options.pageHeading and options.pageCaption %}
+                <h1 class="govuk-heading-l">
+                    <span class="govuk-caption-l">{{ options.pageCaption }}</span>
+                    {{ options.pageHeading }}
+                </h1>
+            {% elseif options.pageHeading %}
+                <span class="govuk-caption-l">{{ options.pageTitle }}</span>
```

Comment:
is the markup intentionally different here to above? If so, I'm not sure I understand why.